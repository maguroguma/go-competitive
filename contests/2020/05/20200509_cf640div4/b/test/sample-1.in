8
10 3
100 4
8 7
97 2
8 8
3 10
5 3
1000000000 9
