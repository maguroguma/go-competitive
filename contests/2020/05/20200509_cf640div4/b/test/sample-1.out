YES
4 2 4
YES
55 5 5 35
NO
NO
YES
1 1 1 1 1 1 1 1
NO
YES
3 1 1
YES
111111110 111111110 111111110 111111110 111111110 111111110 111111110 111111110 111111120
